app/browser/api/ledger.js

```diff
@@ -2234,7 +2234,10 @@ const onCallback = (state, result, delayTime) => {
   // persist the new ledger state
   module.exports.muonWriter(statePath, regularResults)
 
-  run(state, delayTime)
+  // if transactions are present exit the contribution test
+  if (!ledgerState.getInfoProp(state, 'transactions')) {
```

Comment:
@jasonrsadler This should be removed, because what happens if you do second contribution

Reply:
Reworked this to check if the contribution is done. That solution will need to be manually set by the caller to reset for next contribution. We have to have a way to break out to complete the test.